azure-vote-all-in-one-redis.yaml

```diff
@@ -42,9 +53,11 @@ spec:
       labels:
         app: azure-vote-front
     spec:
+      nodeSelector:
+        "beta.kubernetes.io/os": linux
       containers:
       - name: azure-vote-front
-        image: microsoft/azure-vote-front:v1
+        image: privatepelithne.azurecr.io/azuredocs/azure-vote-front:latest
```

Comment:
This should stay a reference to the image in MCR which will appear after a rebase:
https://github.com/Azure-Samples/azure-voting-app-redis/blob/master/azure-vote-all-in-one-redis.yaml#L60